### PR TITLE
DOC: Fix typos and grammar

### PR DESCRIPTION
**Pull request by jhlegarreta**
Fix typos and grammar.

### CODE REVIEW

File: Base/QTGUI/qSlicerFileDialog.h [hunk 1]
```diff
@@ -137,7 +137,7 @@ class Q_SLICER_BASE_QTGUI_EXPORT qSlicerStandardFileDialog
   bool exec(const qSlicerIO::IOProperties& ioProperties =
                     qSlicerIO::IOProperties()) override;
 
-  /// Properties availables with IOPorperties: fileMode, multipleFiles, fileType.
+  /// Properties available with IOPorperties: fileMode, multipleFiles, fileType.
```
Comment (lassoan) [inline Base/QTGUI/qSlicerFileDialog.h:140]:
```suggestion
  /// Properties available with IOProperties: fileMode, multipleFiles, fileType.
```

File: Base/QTCore/qSlicerFileReader.h [hunk 2]
```diff
@@ -72,7 +72,7 @@ class Q_SLICER_BASE_QTCORE_EXPORT qSlicerFileReader
   /// It can be used to determine how specifically extension matched.
   QStringList supportedNameFilters(const QString& fileName, int* longestExtensionMatchPtr = nullptr)const;
 
-  /// Properties availables : fileMode, multipleFiles, fileType.
+  /// Properties available : fileMode, multipleFiles, fileType.
```
Comment (lassoan) [inline Base/QTCore/qSlicerFileReader.h:75]:
```suggestion
  /// Properties available: fileMode, multipleFiles, fileType.
```

File: Libs/MRML/CLI/vtkMRMLCommandLineModuleNode.h [hunk 3]
```diff
@@ -209,8 +209,8 @@ class VTK_MRML_CLI_EXPORT vtkMRMLCommandLineModuleNode : public vtkMRMLNode
   /// \sa AutoRunMode, SetAutoRunMode(), GetAutoRun(), GetAutoRunDelay()
   int GetAutoRunMode()const;
 
-  /// Set the number of msecs to wait before automatically running
-  /// the module. 1000msecs by default.
+  /// Set the delay (in ms) to wait before automatically running
```
Comment (lassoan) [inline Libs/MRML/CLI/vtkMRMLCommandLineModuleNode.h:212]:
```suggestion
  /// Set the delay (in milliseconds) to wait before automatically running
```

**Review by lassoan — COMMENTED** [review]
Perfect, thank you very much!

We have run automated spell checks on the code, but these typos were missed. What software did you use?

**Comment by jhlegarreta** [comment]
> Perfect, thank you very much!

:+1: You're welcome Andras. Thanks for your work with Slicer.

> We have run automated spell checks on the code, but these typos were missed. What software did you use?

I have used [`codespell`](https://github.com/codespell-project/codespell), which was used by Dave Chen and luzpaz to fix some typos across ITK:
https://github.com/InsightSoftwareConsortium/ITK/pull/3921#issue-1580701319

SimpleITK has a GitHub workflow to perform spell checking:
https://github.com/InsightSoftwareConsortium/SimpleITK/blob/2e91a4962f0dd69b95192838ab4c4d1aa657332e/.github/workflows/PythonLintAndSpell.yml#L56

**Comment by lassoan** [comment]
Thank you. We are using codespell, too. Maybe it has not caught the errors because the version or the configuration is different.

**Comment by jhlegarreta** [comment]
I ignore whether there is a way to fix the typos automatically; in my experience, or the way I have used it, it outputs words that are not in its dictionary, and then I have gone through them to manually fix those that I consider are typos.

**Comment by jamesobutler** [comment]
> Maybe it has not caught the errors because the version or the configuration is different.

It appears the Slicer configuration has the following file (Base/QTCLI/vtkSlicerCLIModuleLogic.cxx) in the skip list which would cause codespell not to run there and why changes had to be made here. Reducing the number of skips and keeping the number of exceptions in .codespellignore to a minimum could help.

@jhlegarreta Were you using SimpleITK’s additional dictionary txt file to run with codespell? Or other optional dictionaries?

**Comment by jcfr** [comment]
To clarify, while the script implemented in `SimpleITKSpellChecking` is named `codespell`, it is a completely different implementation.

See https://github.com/SimpleITK/SimpleITKSpellChecking/blob/main/codespell.py

I started to improve it so that:
- it can be integrated using `pre-commit`
- accept some of the same arguments as the "other" codespell project. See https://github.com/codespell-project/codespell

**Comment by jhlegarreta** [comment]
> @jhlegarreta Were you using SimpleITK’s additional dictionary txt file to run with codespell? Or other optional dictionaries?

So I've used SimpleITK's codespell (see @jcfr's comment below). I've used SimpleITK's dictionaries "as is" (have not looked deeply into it, but I assume the tool takes the dictionaries it contains or has access to)/have not specified any additional parameter: I've called the script like `python codespell.py > codespell_slicer.txt`.

> To clarify, while the script implemented in SimpleITKSpellChecking is named codespell, it is a completely different implementation.

True, sorry for the confusion. Thanks for clarifying this @jcfr.

**Comment by jcfr** [comment]
For future reference, the command line interface has been improved to streamline future use in project already integrating with `codespell-project`:
* https://github.com/SimpleITK/SimpleITKSpellChecking/pull/25

Before this can be used in a automatic way, there are ~3500 "false" positive when running the checker.

**Pull request merged**